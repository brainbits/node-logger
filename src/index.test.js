import * as index from './index';

describe('Index', () => {
    it('should export correctly', () => {
        expect(index).toMatchSnapshot();
    });
});
