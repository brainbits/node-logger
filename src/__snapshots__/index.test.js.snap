// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should export correctly 1`] = `
Object {
  "ConnectorError": [Function],
  "CustomError": [Function],
  "HttpError": [Function],
  "logger": Object {
    "log": [Function],
  },
}
`;
